src/js/hora.js

```diff
@@ -335,196 +383,11 @@ define([
             }
         };
 
-        Hora.navigation = {
```

Comment:
Thanks @ericmuyser, @Helen-Mobify looks like the wiki is part of this same repo, so we can just edit it here